src/masonite/mail/drivers/MailgunDriver.py

```diff
@@ -43,11 +43,19 @@ def get_attachments(self):
 
     def send(self):
         domain = self.options["domain"]
+        region = self.options["region"]
```

Comment:
I don't know where my comment went but we need to change this to a `self.options.get("region")`. This is because the region is not going to be set on some people configs